src/GafferSceneUI/ContextAlgo.cpp

```diff
@@ -148,9 +148,9 @@ void expand( Context *context, const PathMatcher &paths, bool expandAncestors )
 	if( needUpdate )
 	{
 		// We modified the expanded paths in place to avoid unecessary copying,
-		// so the context doesn't know they've changed. So we emit the changed
-		// signal ourselves
-		context->changedSignal()( context, g_expandedPathsName );
+		// so the context doesn't know they've changed. So we must let it know
+		// about the change
```

Comment:
I think the previous comment just indicates that I'd overlooked the need to call `changed()` rather than emit `changedSignal()`. I don't think I was trying to do anything clever.